docs:  Continue from the last failed task (#353)
bruin-data/bruin: docs/commands/run.md

| `--push-metadata` | bool | `false` | Push metadata to the destination database if supported (currently BigQuery). |
| `--tag` | str | - | Pick assets with the given tag. |
| `--workers` | int | `16` | Number of workers to run tasks in parallel. |

> |  `--continue` | bool | `false` | Continue from the last failed task. |

Reviewer: ```suggestion
|  `--continue` | bool | `false` | Continue from the last failed asset. |
```

> ### Continue from the last failed task

Reviewer: ```suggestion
### Continue from the last failed asset
```

If you want to continue from the last failed task, you can use the `--continue` flag. This will run the pipeline/asset from the last failed task. Bruin will automatically retrive all the flags used in the last run. 

```bash
bruin run --continue 
```

> NOTE: This will only work if the pipeline structure is not changed. If Pipeline/Asset dependencies are changed, you will need to run the pipeline/asset from the beginning. This is to ensure that the pipeline/asset is run in the correct order.

Reviewer: We can format this a bit nicer:
```suggestion
> [!NOTE]
> This will only work if the pipeline structure is not changed. If the pipeline structure has changed in any way, including asset dependencies, you will need to run the pipeline/asset from the beginning. This is to ensure that the pipeline/asset is run in the correct order.
```
